src/python_testing/TC_DeviceBasicComposition.py

```diff
@@ -674,6 +674,25 @@ class RequiredMandatoryAttribute:
             self.fail_current_test(
                 "At least one cluster has failed the range and support checks for its listed attributes, commands or features")
 
+        # Wildcard reads of events: event list MUST NOT be empty since at least a boot event should be registered.
+        self.print_step(12, "Validate that event wildcard subscription works")
```

Comment:
```suggestion
        self.print_step(12, "Validate that event wildcard subscription works does not fail to initialize.")
```

Reply:
The test is intended to test for a subset of events working, rather than just saying "assert not throws".